src/main/java/uk/gov/hmcts/dts/fact/entity/AreaOfLaw.java

```diff
@@ -3,10 +3,7 @@
 import lombok.Getter;
 import lombok.Setter;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
I would normally go for the individual imports as before, the wildcard here may cause an issue in sonar

Reply:
Ahh yes, thanks Shaun!